File: app/src/main/java/com/pkfcooperparry/template/data/model/db/visit/Value.java
package com.pkfcooperparry.template.data.model.db.visit;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import io.realm.RealmObject;
import lombok.Getter;
import lombok.Setter;

public class Value extends RealmObject {
    @Getter
    @Setter
    @SerializedName("idValue")
    @Expose
    private Long idValue;
    @Getter
    @Setter
    @SerializedName("flatValue")
    @Expose
    private String flatValue;
    @Getter
    @Setter
    @SerializedName("dataValue")
    @Expose
    private Long dataValue;


    public Value() {
    }
}

File: app/src/main/java/com/pkfcooperparry/template/data/model/db/visit/Value.kt
package com.pkfcooperparry.template.data.model.db.visit

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

import io.realm.RealmObject
import lombok.Getter
import lombok.Setter


open class Value : RealmObject() {

    @SerializedName("idValue")
    @Expose
    var idValue: Long? = null

    @SerializedName("flatValue")
    @Expose
    var flatValue: String? = null

    @SerializedName("dataValue")
    @Expose
    var dataValue: Long? = null
}
